setUp.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+clear
+echo "Initializing sys variables..."
+
+export TWITTER_CONSUMER_KEY=KXjY39ROD2QMa0LUIkEBSnJMM
+export TWITTER_CONSUMER_SECRET=lX27EskdFXqFCYwQCo7zK8c7FT9NgL2YpDox0anq4U9g6SrOKG
+export TWITTER_ACCESS_TOKEN=722849671593402368-Fx0XHOaSsIyCQy0VJ6ZWnZ97TmWJ3CG
+export TWITTER_ACCESS_SECRET=ByBD930AcAK9Ue00PvWXMFQ3o7j4RSRSZXthz0rigzKOk
+
+echo "Done. Accessing Twitter api should work now..."
```

Comment:
This is essentially the same as exposing your keys ... you shouldn't commit this file into source control. Please utilize the `.gitignore`